client/tests/integration/events/pipeline.rs

```diff
@@ -100,7 +100,7 @@ impl Checker {
 
 #[test]
 fn committed_block_must_be_available_in_kura() {
-    let (_rt, peer, client) = <PeerBuilder>::new().with_port(10_985).start_with_runtime();
+    let (_rt, peer, client) = <PeerBuilder>::new().with_port(11_040).start_with_runtime();
```

Comment:
```suggestion
    let (_rt, peer, client) = <PeerBuilder>::new().with_any_port().start_with_runtime();
```

Would be nice to have a `#[cfg(test)]` method `::with_any_port()` which find any available port on a machine.

Related:

- https://github.com/hyperledger/iroha/issues/2137

Reply:
@0x009922 initially we used such approach with [unique_port](https://github.com/soramitsu/iroha2-unique_port) crate, unfortunately it didn't work very well because there were no ipc and accidentally 2 different tests were trying to take the same port. 
When we run tests they are run in different processes and we need some form of ipc to distribute available ports without large overhead.